include/dbus_privileges.hpp

```diff
@@ -90,10 +94,22 @@ inline bool
         asyncResp->res.result(boost::beast::http::status::forbidden);
         if (req.session->isConfigureSelfOnly)
         {
-            redfish::messages::passwordChangeRequired(
-                asyncResp->res,
-                boost::urls::format("/redfish/v1/AccountService/Accounts/{}",
-                                    req.session->username));
+            if (req.session->isGenerateSecretKeyRequired)
+            {
```

Comment:
does it make sure order of throwing passwordChangeRequired first? when both passwordChangeRequired and isGenerateSecretKeyRequired set?